test/integration/api/nep5.js

```diff
@@ -47,6 +47,24 @@ describe('Integration: API NEP5', function () {
       })
   })
 
+  it('getTokenBalances', () => {
+    return NEP5.getTokenBalances(
+      DEFAULT_RPC.TEST,
+      [
+        CONTRACTS.TEST_RPX,
+        CONTRACTS.TEST_RHTT4,
+        CONTRACTS.TEST_LWTF,
+        CONTRACTS.TEST_NXT
+      ],
+      testKeys.b.address
+    ).then(result => {
+      result.RPX.should.equal(0)
+      result.RHTT4.should.equal(2)
+      result.LWTF.should.equal(0.00000746)
+      result.NXT.should.equal(0)

```

Comment:
Let us check that the results returned should be have the same length as the array of scripthashes. We cannot assert that the balances will always remain the same else the test will be too fragile.

Reply:
Ok cool. will change it shortly.